gesamt/jalaka_05.0

```diff
@@ -0,0 +1,4 @@
+//test if flat_closure by parameter works
+my_call(x)
+    return (x+16) @(5);
```

Comment:
Damn this is a trickly line.
Maybe we can add a description that the calling C Code substracted 16 from the function pointer so we have to correct it here?